rust/xnvme-sys/Cargo.toml

```diff
@@ -1,10 +1,18 @@
 [package]
 name = "xnvme-sys"
-version = "0.0.0"
+version = "0.7.1"
```

Comment:
It looks like we are locking version of Rust bindings to xnvme version. Does xnvme use proper semver? If we find a bug in the Rust bindings and we need to bump patch version for the Rust crate, will we do a release of C xnvme with same patch level bump?

Reply:
yup, xNVMe subscribes to the dogma of SemVer. That is why xNVMe is currently at < 1.0, to have the freedom granted by paragraph 4 and 5 of SemVer.

Yup, I find that locking the ``xnvme-sys`` crate to the xnvme version is sensible, since it is such as "raw" binding to the xNVMe library, and yes, the intent is to bump xNVMe version in case issues are found in the Rust bindings. This also holds for the Python bindings.

For the potential ``xnvme`` create, e.g. the idiomatic-bindings, then this might have its own versioning-number, since it will introduce Rust-specific and self-contained constructs. E.g. once xNVMe itself hits v1, v2 etc. then the idiomatic bindings might not be ready for that...